source/PlayerInfoPanel.h

```diff
@@ -16,8 +16,13 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include "Panel.h"
 
 #include "ClickZone.h"
+#include "GameData.h"
+#include "InfoPanelState.h"
 #include "Point.h"
+#include "text/Table.h"
```

Comment:
We can move this back to the .cpp file, now that we don't need it for the `Alignment` enum. (A `text/layout.hpp` include will be needed instead.)